src/Symfony/Component/HttpKernel/DataCollector/RequestDataCollector.php

```diff
@@ -59,12 +59,30 @@ public function collect(Request $request, Response $response, \Exception $except
 
         $requestFiles = array();
         foreach ($request->files->all() as $files) {
+            if (null === $files) {
+                continue;
+            }
+
+            if (!\is_array($files)) {
+                $files = array($files);
+            }
+
             foreach ($files as $fileName => $fileData) {
-                $requestFiles[] = array(
-                    'name' => $fileData->getClientOriginalName(),
-                    'mimetype' => $fileData->getMimeType(),
-                    'size' => $fileData->getSize(),
-                );
+                if (null === $fileData) {
+                    continue;
+                }
+
+                if (!is_array($fileData)) {
+                    $fileData = array($fileData);
```

Comment:
fixes `Call to a member function getClientOriginalName() on array` for nested multiple form.

Reply:
ouch.. I think you're right :D

Reply:
fixed